tasks/libs/common/utils.py

```diff
@@ -42,6 +43,19 @@
 AGENT_VERSION_CACHE_NAME = "agent-version.cache"
 
 
+@dataclass
+class TimedOperationResult:
+    path: str
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏For the linter it's a path, but if it's meant to be used in other tasks then you could just use "name"